Lib/test/test_datetime.py

```diff
@@ -20,7 +20,7 @@
 # XXX(gb) First run all the _Pure tests, then all the _Fast tests.  You might
 # not believe this, but in spite of all the sys.modules trickery running a _Pure
 # test last will leave a mix of pure and native datetime stuff lying around.
-test_classes = []
+all_test_classes = []
```

Comment:
Hmm, so the value used in as `str(class)` is actually `cls.__qualname__`. Upon setting the correct name, at least one other test fails, which conditioned on there being `_Fast` being in `str(self)` before running. I'm now looking at how to fix that test.